hooks/useApproval.ts

```diff
@@ -57,6 +57,7 @@ const useApproval = (
     'allowance',
     {
       args: [address || ethers.constants.AddressZero, spenderAddress],
+      enabled: false,
```

Comment:
https://wagmi.sh/docs/hooks/useBlockNumber#enabled-optional

Reply:
> Set this to false to disable this query from automatically running.

Read runs automatically! If I set it to false, is the behavior like the first time I do a refetch, it gets it?

Reply:
If I understand correctly, the default is to perform a query to the contract at mount time. This is the desired behavior if it is at wallet connection.

In this case, when the wallet is not connected, a query is made to the contract and an error occurs, so I think I have changed it so that a query is made to the contract at the time of the hooks call by setting the enabled option false.